src/tutorials/volume-snapshots-how-to-create-and-use-data-clones-in-okteto.mdx

```diff
@@ -0,0 +1,311 @@
+-   Familiarity with **Kubernetes manifests**
+-   [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine
+
+## Step 1: Deploy the Sample Application
+
+For this tutorial, we’ll deploy a CRUD recipe application that stores data in MongoDB.
+
+Clone the repository:
+```console
+$ git clone https://github.com/okteto/fastapi-snapshot-app
+$ cd fastapi-snapshot-app
+```
+
+In the `manifests` folder, the `k8s.yml` manifest file is responsible for the deployment of the application's service, and the `mongo.yml` manifest file is responsible for deploying the database service. The database service in this case is a MongoDB service.
+
+Apply the manifests to deploy the application:
+
+```console
+okteto kubeconfig
+kubectl apply -f manifests/
+```
+
+Expected output:
+```console
+deployment.apps/fastapi-snapshot-demo created
+service/fastapi-snapshot-demo created
+persistentvolumeclaim/data created
+service/mongodb created
+statefulset.apps/mongodb created
+```
+
+<Image
+  src={require("@site/static/img/tutorials/volume-snapshots/deployed.png").default}
+  alt="Deployed Application"
+  width="1000"
+/>
+
+✅ Your application is now running in Okteto. Take note of the **endpoint URL** under **Endpoints** in the Okteto Dashboard.
+
+
+## Step 2: Populate the Application Database
+
+Before taking a snapshot, add sample data. This is the data that we'll be including in the Development Environments that we'll create in the next steps.
+
+```console
+curl -X 'POST' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "Donuts",
+  "ingredients": ["Flour", "Milk", "Sugar", "Vegetable Oil"]
+}'
+```
+
+```console
+curl -X 'POST' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "Pancake",
+  "ingredients": ["Flour", "Milk", "Sugar", "Baking Powder", "Vegetable Oil"]
+}'
+```
+
+Verify that the database has been populated by retrieving the list of recipes present:
+
+```console
+ curl -X 'GET' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json'
+```
+
+Expected response:
+
+```json
+{
+  "data": [
+    {
+      "id": "60ce68651eeddf1c5ab796d1",
+      "name": "Donuts",
+      "ingredients": [
+        "Flour",
+        "Milk",
+        "Sugar",
+        "Vegetable Oil"
+      ]
+    },
+    {
+      "id": "60ce689c1eeddf1c5ab796d2",
+      "name": "Pancake",
+      "ingredients": [
+        "Flour",
+        "Milk",
+        "Sugar",
+        "Baking Powder",
+        "Vegetable Oil"
+      ]
+    }
+  ]
+}
+```
+
+## Step 3: Create a Database Snapshot
+
+A Volume Snapshot will capture the database state at this moment.
+
+Create a new manifest file `snapshot.yml` in the `manifests` folder and add the following to it:
```

Comment:
this is currently available at the root of the repo, its  the `vol.yaml` file (I think). should we remove it? (or maybe we keep a separate branch with all the completed manifests?